rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,134 @@
+# RFC: Communicating new package structure for `@fluentui/*`
+
+**Note that this RFC focuses solely on the package structure from a consumer perspective, focused primarily on v7/8 and converged packages. It does not deal with folder structure or northstar packages.**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+Currently, we have a lot of different component packages, and it's not clear to consumers what the purpose of each one is or whether it's production-ready.
+
+There are a few aspects to the proposed solution:
+
+- Add badges (or similar) to package readmes to clarify GA/preview/dev status
+- "Converged" component packages only contain convergence work
+  - Move old (v8) component implementations from `react-{checkbox,link,slider,tabs,toggle}` back to `react-internal`
+  - Move prototype "converged" components to the package roots and get rid of `/next` folders
+  - Probably also move all component implementations in `react` back to `react-internal`
+    - OR (open question) just get rid of `react-internal`?
```

Comment:
Yes to this. This would remove complexity from our project.

Reply:
Updating this to say we should just get rid of `react-internal`. It does create some issues with `react-date-time` but this can be dealt with.